types/kafka-node/index.d.ts

```diff
@@ -101,11 +102,34 @@ export class KeyedMessage {
 }
 
 export class Admin {
-    constructor(kafkaClient: KafkaClient);
-    listGroups(cb: (error: any, data: any) => any): void;
-    describeGroups(consumerGroups: any, cb: (error: any, data: any) => any): void;
+         constructor(kafkaClient: KafkaClient);
```

Comment:
Can you please use 4-space indentation?

Reply:
sorry, I had a misconfigured ws, now it should be fixed.